packages/devtools_app/lib/src/screens/memory/panes/control/primary_controls.dart

```diff
@@ -19,6 +19,7 @@ class PrimaryControls extends StatelessWidget {
       showChart: preferences.memory.showChart,
       onPressed: (show) => preferences.memory.showChart.value = show,
       minScreenWidthForTextBeforeScaling: memoryControlsMinVerboseWidth,
```

Comment:
can we also add "Memory usage" as a label here? We can make the tooltip more verbose: "Toggle the visibility of the Memory usage chart". Same comment below for the tooltip on the Performance screen "Toggle the visibility of the Flutter frames chart"